sbi/inference/trainers/npe/mnpe.py

```diff
@@ -32,7 +36,11 @@ class MNPE(NPE_C):
     def __init__(
         self,
         prior: Optional[Distribution] = None,
-        density_estimator: Union[Literal["mnpe"], DensityEstimatorBuilder] = "mnpe",
+        density_estimator: Union[
+            Literal["mnpe"],
+            DensityEstimatorBuilder[ConditionalVectorFieldEstimator],
```

Comment:
Same as in `MNLE`, 'MNPE` is a special case where the vector fields do not work. please remove.